packages/http-smoke-testing/.travis.yml

```diff
@@ -17,13 +17,14 @@ matrix:
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
         -   php: 7.1
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
+        -   php: 7.1
```

Comment:
so there will be 3 tests for 7.1 now ?
so if the `php` section will be removed and distributed into `matrix include` it will be easier and more understandable

Reply:
Maybe, I'll try that :+1:

Reply:
Done in 36a2c6b4b, I've inlined the objects because it was hard to read when in some jobs the `env` was missing. I think it's better now